seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -450,7 +450,7 @@
       ],
       "tests": [
         "assert(myVar === 10, 'message: <code>myVar</code> should equal <code>10</code>');",
-        "assert(/myVar[-]{2}/.test(code), 'message: Use the <code>--</code> operator on <code>myVar</code>');",
+        "assert(/myVar\\s*\\-{2}/.test(code), 'message: Use the <code>--</code> operator on <code>myVar</code>');",
```

Comment:
@SaintPeter You're correct that `-` is not a special character and does not need to be escaped.